### DISCUSSION TITLE
PopoverTrigger in tsx show this

### DISCUSSION THREAD

**Discussion by hakimLyon** [chakra-ui/chakra-ui#5897] [Q&A] [answered]
The type '{ children: Element; }' has no properties in common with the type 'IntrinsicAttributes'.ts(2559)
Please How to fix that?

**Comment by eckertalex** [comment, marked as answer]
Issue is already reported here #5896 

I created a [CodeSandbox](https://codesandbox.io/s/chakra-discussion-popover-trigger-1db2px?file=%2Fsrc%2Fapp.tsx) for this.
